packages/material-ui/src/ExpansionPanelSummary/ExpansionPanelSummary.js

```diff
@@ -128,7 +128,7 @@ class ExpansionPanelSummary extends React.Component {
         disableRipple
         disabled={disabled}
         component="div"
-        aria-expanded={expanded}
+        aria-expanded={Boolean(expanded)}
```

Comment:
What's the need for this change?
```js
expanded: PropTypes.bool,
```

Reply:
If it's passed as `undefined` it won't appear in the DOM not indicating that the element expands some content.

Reply:
In practice, it should always be defined:
https://github.com/mui-org/material-ui/blob/64013d6c817b3c9bf2bd006aee2ac3f4a948a01f/packages/material-ui/src/ExpansionPanel/ExpansionPanel.js#L135-L137

Reply:
Are we arguing whether we should put this in? Why not be safe and always have it as a boolean. It's explicit, helps test. Could you elaborate what the downsides are here?

Reply:
What I'm saying is that we can be "smart" about it. We know that in practice, the `Boolean()` will be equivalent to an identity function. I believe the tests could always provide a boolean value if we want to match the reality.

Reply:
You're right. I'm removing this. I think it might be a better idea to move those controlled props into context anyway. Gets rid of cloneElement, allows a fragment as a child and we can improve the DX when by setting the default value of `expanded` to a boolean. That's for another PR though